video/out/hwdec/hwdec_cuda.c

```diff
@@ -57,12 +57,12 @@ int check_cu(const struct ra_hwdec *hw, CUresult err, const char *func)
 
 #define CHECK_CU(x) check_cu(hw, (x), #x)
 
-static const cuda_interop_init interop_inits[] = {
+static const cuda_interop_fn *interop_fns[] = {
 #if HAVE_GL
-    cuda_gl_init,
+    &(static cuda_interop_fn) {.check = cuda_gl_check, .init = cuda_gl_init},
 #endif
 #if HAVE_VULKAN
-    cuda_vk_init,
+    &(static cuda_interop_fn) {.check = cuda_vk_check, .init = cuda_vk_init},
 #endif
     NULL
 };
```

Comment:
The syntax you are using here looks to be gcc specific. I'm not sure what about it is non-standard, but it's failing the clang and windows builds.

Reply:
It should work fine without `static` keyword.

Reply:
The storage-class-specifier in compound literals is a C23 feature.

[1] https://en.cppreference.com/w/c/language/compound_literal
[2] https://en.cppreference.com/w/c/compiler_support/23

Reply:
Yep, we are C11 in mpv world.

Reply:
I think this should be fixed now? Fingers crossed